### PR TITLE
add is-confidentiality-protected flag

### PR DESCRIPTION
**Pull request by nedmsmith**
added is-confidentiality-protected flag to flags-map.
Fixes issue #101

### CODE REVIEW

File: cddl/examples/comid-flags.diag [hunk 1]
```diff
@@ -39,7 +39,8 @@
                 / integrity-protected / 5 : true,
                 / runtime-meas / 6 : true,
                 / immutable / 7 : true,
-                / tcb / 8 : true
+                / tcb / 8 : true,
+                / conf-protected / 9 : true
```
Comment (yogeshbdeshpande) [inline cddl/examples/comid-flags.diag:43]:
```suggestion
                / confidentiality-protected / 9 : true
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -989,6 +989,10 @@ after being loaded into memory.
 * `is-tcb` (index 8): If the flag is true, the measured environment is a trusted
 computing base.
 
+* `is-confidentiality-protected` (index 9): If the flag is true, the measured environment
+is confidentiality protected. For example, if the measured environment consists of memory,
+the sensitive values in memory are obfuscated, such as, by encryption.
```
Comment (andrew-draper) [inline draft-ietf-rats-corim.md]: "such as," might be better worded as "for example"

Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:994]:
```suggestion
the sensitive values in memory are encrypted.
```

**Review by andrew-draper — APPROVED** [review]

**Pull request merged**